mail/src/test/java/com/sun/mail/util/logging/MailHandlerTest.java

```diff
@@ -450,33 +450,15 @@ private void testLoggable(Level lvl, LogRecord record) {
         instance.setErrorManager(em);
 
         instance.setLevel(lvl);
-        MemoryHandler mem = null;
         boolean result = false;
         boolean expect = true;
-        try {
-            result = instance.isLoggable(record);
-            mem = new MemoryHandler(new ConsoleHandler(), 100, Level.OFF);
-            mem.setErrorManager(em);
-            mem.setLevel(lvl);
-            expect = mem.isLoggable(record);
-        } catch (RuntimeException mailEx) {
-            try {
-                if (mem != null) {
-                    fail("MemoryHandler threw and exception: " + mailEx);
```

Comment:
threw *an* exception

Reply:
Correct.  Good thing is that typo is corrected because the code is deleted :)

Reply:
D'oh!